CHANGELOG

```diff
@@ -0,0 +1,109 @@
+# Changelog
+
+## [Unreleased]
+### New Features
+ * Support reapply configuration without link down when possible.
+ * Relicensed to LGPL 2.1+.
+ * Allow editing existing interface without type property in desire state.
+
+### Bug Fixes
+
+ * Allow adding slave to linux bridge with name only.
+ * Allow enabling IPv6 with empty address.
+ * netapplier: Initialize mainloop global object on failure.
+ * DNS: Preserve DNS config with empty state.
+
+## [0.0.7] - 2019-06-13
+### New Features
+ * Support editing DNS client configuration.
+ * Support editing IP routes.
+ * API: Promote the public API to libnmstate level:
+    * libnmstate.show()
+    * libnmstate.apply()
+    * libnmstate.commit()
+    * libnmstate.rollback()
+    * libnmstate.error
+    * libnmstate.schema
+
+### Bug Fixes
+
+ * netapplier: Optimize new and existing interface processing.
+
+
+## [0.0.6] - 2019-04-24
+### New Features
+ * Support manual transaction control.
+ * Support querying DNS client configuration.
+ * Support querying IP routes.
+ * Support changing MAC address.
+
+### Bug Fixes
+ * nmstatectl: Handle state editing conflict.
+ * netapplier: Handle commit/rollback errors.
+ * nm.checkpoint: Handle DBusExceptions.
+ * Fix build on old setuptools.
+ * nm: Exit with a human readable error when missing libnm introspection.
+ * state: Rebuild the ifaces state on each call to the state getter.
+ * metadata: New created iface entry should include a name.
+ * state: Relocate the remove func of absent/down ifaces into State.
+ * netapplier: Pass the State object to the metadata generator.
+
+## [0.0.5] - 2019-03-12
+### New Features
+ * Support Open vSwitch internal interface.
+ * Support ignoring DNS/routes/gateways from DHCP/Autoconf.
+
+### Bug Fixes
+ * nm device: Include safety checks when deleting a device.
+ * nm: Delete explicitly OVS type interfaces.
+ * netinfo: Do not include the mtu entry for ovs-bridge.
```

Comment:
How about changing from `netinfo` =>> `show`?

Generally, I would prefer not to put implementation details like the module or class names in the changelog. I prefer to see here the part that affects the user of the lib in a language they understand.

Reply:
That would require a lot effort to explain commits from code to user friendly message.  
Can we only do it for new bug fix message in changelog?

Reply:
I'm good with focusing on user readability for new items only, but some points can be easily change. Like `netinfo` to `show`. Anytime we referenced something internal, just give its public name.